server/routes/functionalSkills/functionalSkillsController.ts

```diff
@@ -17,10 +17,17 @@ export default class FunctionalSkillsController {
     const latestFunctionalSkillsFromCurious = mostRecentFunctionalSkills(functionalSkillsFromCurious)
     const allFunctionalSkillsFromCurious = allFunctionalSkills(functionalSkillsFromCurious)
 
+    const englishSkills = functionalSkillsByType(functionalSkillsFromCurious.assessments, 'ENGLISH')
+    const mathsSkills = functionalSkillsByType(functionalSkillsFromCurious.assessments, 'MATHS')
+    const digitalSkills = functionalSkillsByType(functionalSkillsFromCurious.assessments, 'DIGITAL_LITERACY')
+
     const view = new FunctionalSkillsView(
       prisonerSummary,
       latestFunctionalSkillsFromCurious,
       allFunctionalSkillsFromCurious,
+      englishSkills,
```

Comment:
OK, you're gonna hate me for not mentioning this before (but I only remembered last night), but we're going to need one more field in the view class `FunctionalSkillsView`
We need `problemRetrievingData: boolean` ..... because currently it exists in the 2 fields `latestFunctionalSkillsFromCurious` and `allFunctionalSkillsFromCurious` ; and the nunjucks view reads it from `allFunctionalSkillsFromCurious.problemRetrievingData`

But ... shortly (in a couple of PRs time), you're going to remove these 2 fields from the view class

So, I think you want a new `problemRetrievingData: boolean` on `FunctionalSkillsView`, and you'll provide its value here on the constructor call from `functionalSkillsFromCurious.problemRetrievingData`

Happy to talk it through if I've not made much sense here 😁

Reply:
I've added this; let me know what you think of the eslint error fix approach though in the below comment.